health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/configs/UICustomizations.js

```
@@ -218,33 +219,54 @@ export const UICustomizations = {
+            }
+            catch(error){
+              setShowToast({label: error?.response?.data?.Errors?.[0]?.message, type: "error"})
+              return error;
+            }
           }
           const downloadExcelTemplate = async () => {
             // const res = await generateFile();
             // await delay(2000);
             const resFile = await generateTemplate();
+            
             if (resFile && resFile?.GeneratedResource?.[0]?.fileStoreid) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)